etcd-configuration.adoc

```diff
@@ -0,0 +1,51 @@
+// vim: tw=79
+= etcd installation and configuration for Tendrl
+
+== etcd
+etcd is an open-source distributed, consistent key value store that provides a
+reliable way to store and access the data for shared configuration and service
+discovery across a cluster. Tendrl is using etcd to store the yaml files, other
+configuration files and its data.
+
+== Installation
+etcd can be installed in any server. However for tendrl application, it can be
+installed in the server where tendrl-api is installed. The package etcd can be
+installed by running "yum install etcd"
```

Comment:
should we mention from which repo is etcd available like epel for centOS and for other supported upstream platforms.